templates/cluster-stamp.json

```diff
@@ -48,6 +48,20 @@
                 "description": "IP ranges authorized to contact the Kubernetes API server. Passing an empty array will result in no IP restrictions. If any are provided, remember to also provide the public IP of the egress Azure Firewall otherwise your nodes will not be able to talk to the API server (e.g. Flux)."
             }
         },
+        "asbDomain": {
```

Comment:
Min length seems long. a.ab.cd is valid

Max length seems short. Maybe 64? 128?  I think 255 is the RFC max but not sure.

Reply:
done!